egs/wsj/s5/utils/combine_ali_dirs.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+# Copyright 2016  Xiaohui Zhang  Apache 2.0.
+
+# This script operates on alignment directories, such as exp/tri4a_ali
+
+# Begin configuration section. 
+cmd=run.pl
+extra_files= # specify addtional files in 'src-data-dir' to merge, ex. "file1 file2 ..."
+ref_data_dir= # data directory to be used as reference for rearranging alignments
+num_jobs= # number of jobs used to split the reference data directory.
+# End configuration section.
+set -x
+echo "$0 $@"  # Print the command line for logging
+
+if [ -f path.sh ]; then . ./path.sh; fi
+. parse_options.sh || exit 1;
+
+if [[ $# -lt 2 || ! "$ref_data_dir" || ! "$num_jobs" ]]; then
+  echo "Usage: combine_ali_dirs.sh --ref-data-dir data/train --num-jobs 32 [--extra-files 'file1 file2'] <dest-ali-dir> <src-ali-dir1> <src-ali-dir2> ..."
+  echo "Note, files that don't appear in the first source dir will not be added even if they appear in later ones."
```

Comment:
There is no e.g. line. Usage line is mixed with e.g. line.
--num-jobs 32 must go to the e.g line. The usage line must usually not have values in them.